pkg/windows/windows.go

```diff
@@ -800,6 +800,7 @@ func (vm *windows) removeHNSNetworks() error {
 	cmd := "\"$net0 = Get-HnsNetwork | where { $_.Name -eq '" + BaseOVNKubeOverlayNetwork + "'};" +
 		"$net1 = Get-HnsNetwork | where { $_.Name -eq '" + OVNKubeOverlayNetwork + "'};" +
 		"$net0 | Remove-HnsNetwork;" +
+		"Start-Sleep -s 10;" +
```

Comment:
This warrants a comment in the code too. I however don't understand the fix. How can the command return before executing fully? By the same logic isn't it possible for the command to return without executing the sleep? 

I am also wondering if the order of deletion is important. Do you know deleting which network causes a blip in the connectivity?

I suspect you are running into issues because the network is getting reconfigured. You might have to split the call into separate commands and re-initialize the SSH connection in between.

Reply:
Agree with your statement. My earlier thought process was the sleep might give time for the first network to cleanup before executing the next command.
Removing the second network  OVNKubeOverlayNetwork causes loss in connectivity. That is why it is the last one being removed.
Separating into two different calls seems to be the best solution. I made the changes for that. Testing out locally to see if the issue persists.